mongo/database.go

```diff
@@ -192,8 +192,9 @@ func (db *Database) RunCommand(ctx context.Context, runCommand interface{}, opts
 	}
 
 	err = op.Execute(ctx)
+	_, convErr := processWriteError(err)
```

Comment:
Add a comment explaining why we're using processWriteError rather than replaceErrors even though this function doesn't execute a write.